src/Pos/Explorer/Web/ClientTypes.hs

```diff
@@ -242,14 +242,15 @@ data CAddressType =
     | CScriptAddress
     | CRedeemAddress
     | CUnknownAddress
-    deriving (Show, Generic)
+    deriving (Show, Generic, Eq)
 
 data CAddressSummary = CAddressSummary
-    { caAddress :: !CAddress
-    , caType    :: !CAddressType
-    , caTxNum   :: !Word
-    , caBalance :: !CCoin
-    , caTxList  :: ![CTxBrief]
+    { caAddress    :: !CAddress
+    , caType       :: !CAddressType
+    , caTxNum      :: !Word
+    , caBalance    :: !CCoin
+    , caTxList     :: ![CTxBrief]
+    , caIsRedeemed :: !(Maybe Bool)
```

Comment:
Done. Now we have something like this:
![address-detail-with-isredeemed](https://user-images.githubusercontent.com/47693/28520220-ef177264-706e-11e7-9b15-e8596693c35b.png)